drivers/net/ethernet/renesas/rswitch.h

```diff
@@ -285,7 +285,7 @@ struct rswitch_private {
 	u32 desc_bat_size;
 	phys_addr_t dev_id;
 
-	struct rswitch_device *rdev[RSWITCH_MAX_NUM_NDEV];
+	struct list_head rdev_list;
```

Comment:
I see in how many context this list is used. I think it should be used while holding some lock to prevent shared resource issues.